controllers/dexserver_controller.go

```diff
@@ -980,7 +995,27 @@ func (r *DexServerReconciler) syncConfigMap(dexServer *authv1alpha1.DexServer, c
 					NameAttr:     connector.LDAP.GroupSearch.NameAttr,
 				}
 			}
+		case authv1alpha1.ConnectorTypeOIDC:
+			// Get Github ClientSecret from SecretRef
+			clientSecret, err := getConnectorSecretFromRef(connector, dexServer, r, ctx)
 
+			if err != nil {
+				log.Error(err, "Error getting client secret")
+				return err
+			}
+
+			newConnector = DexConnectorSpec{
+				Type: string(authv1alpha1.ConnectorTypeOIDC),
+				Id:   connector.Id,
+				Name: connector.Name,
+				Config: DexConnectorConfigSpec{
+					ClientID:     connector.OIDC.ClientID,
+					ClientSecret: clientSecret,
```

Comment:
Just an FYI - I realized that https://github.com/identitatem/dex-operator/pull/95 is not merged yet and fell off my radar 😞 . Sensitive values like client secret need to be provided to the configMap as environment variables. Just a heads up that this may need to be updated for the OIDC connector when https://github.com/identitatem/dex-operator/pull/95 is merged.

Reply:
Yeah I'll update for oidc once #95 is merged